scenes/MosaikHeer.scn

```diff
@@ -278,148 +278,148 @@
 /ch/10/gate/filter OFF 3.0 990.9
-/ch/10/dyn ON COMP PEAK LOG -18.5 5.0 2 3.00 10 10.0  30 POST 0 100 OFF
-/ch/10/dyn/filter OFF 3.0 990.9
+/ch/10/dyn ON COMP PEAK LOG -16.0 2.5 4 2.00 10 10.0  151 POST 0 100 ON
+/ch/10/dyn/filter ON 1.0 3k31
 /ch/10/insert OFF POST OFF
 /ch/10/eq ON
-/ch/10/eq/1 PEQ 133.7 -1.00 2.0
-/ch/10/eq/2 PEQ 2k89 +1.50 1.0
-/ch/10/eq/3 PEQ 4k68 +2.25 0.9
-/ch/10/eq/4 HShv 10k02 +2.75 2.0
-/ch/10/mix ON  -5.2 OFF +0 OFF  -4.3
-/ch/10/mix/01 ON  -0.5 +0 PRE
-/ch/10/mix/02 ON  -0.5
+/ch/10/eq/1 PEQ 124.7 +0.00 2.0
+/ch/10/eq/2 PEQ 701.5 -4.25 1.3
+/ch/10/eq/3 PEQ 1k97 -5.25 1.3
+/ch/10/eq/4 HShv 5k20 -2.75 2.0
+/ch/10/mix ON   -oo ON +0 OFF  -3.8
+/ch/10/mix/01 ON   -oo +0 PRE
+/ch/10/mix/02 ON   -oo
 /ch/10/mix/03 ON   -oo +0 PRE
 /ch/10/mix/04 ON   -oo
-/ch/10/mix/05 ON   -oo +0 PRE
-/ch/10/mix/06 ON   -oo
+/ch/10/mix/05 ON   0.0 +0 PRE
+/ch/10/mix/06 ON   0.0
 /ch/10/mix/07 ON   -oo +0 PRE
 /ch/10/mix/08 ON   -oo
 /ch/10/mix/09 ON   -oo +0 PRE
 /ch/10/mix/10 ON   -oo
-/ch/10/mix/11 ON  +1.5 -48 POST
-/ch/10/mix/12 ON  +1.5
+/ch/10/mix/11 ON  -1.5 +0 POST
+/ch/10/mix/12 ON  -1.5
 /ch/10/mix/13 ON   -oo +0 POST
-/ch/10/mix/14 ON   -oo
+/ch/10/mix/14 ON -10.0
 /ch/10/mix/15 ON   -oo +0 POST
 /ch/10/mix/16 ON   -oo
-/ch/10/grp %00000001 %000000
+/ch/10/grp %00000010 %000000
 /ch/10/automix OFF  +0.0
-/ch/11/config "A Git" 1 GN 11
+/ch/11/config "E Bass" 1 GN 11
 /ch/11/delay OFF   0.3
-/ch/11/preamp -18.0 OFF ON 24 121
-/ch/11/gate OFF GATE 0.0 60.0 1  502  983 0
+/ch/11/preamp -18.0 OFF ON 24  51
+/ch/11/gate OFF GATE -80.0 60.0 1  502  983 0
 /ch/11/gate/filter OFF 3.0 990.9
-/ch/11/dyn ON COMP PEAK LOG -28.0 3.0 2 3.00 16 10.0  151 POST 0 100 ON
-/ch/11/dyn/filter OFF 3.0 990.9
-/ch/11/insert OFF POST OFF
+/ch/11/dyn ON COMP PEAK LOG -15.5 4.0 3 4.00 32 0.02  18 POST 0 100 OFF
+/ch/11/dyn/filter ON 1.0 101.4
+/ch/11/insert OFF POST FX6R
 /ch/11/eq ON
-/ch/11/eq/1 PEQ 170.2 -1.25 1.9
-/ch/11/eq/2 PEQ 295.8 -3.00 2.2
-/ch/11/eq/3 PEQ 2k11 -3.75 1.1
-/ch/11/eq/4 HShv 9k68 +2.50 2.0
-/ch/11/mix ON   -oo ON +0 OFF  +1.3
+/ch/11/eq/1 PEQ 71.8 +5.25 1.7
+/ch/11/eq/2 PEQ 209.4 -6.00 1.9
+/ch/11/eq/3 PEQ 590.2 -8.25 8.6
+/ch/11/eq/4 PEQ 833.7 +8.00 1.3
+/ch/11/mix ON   -oo ON +0 ON  -1.5
 /ch/11/mix/01 ON   -oo +0 PRE
 /ch/11/mix/02 ON   -oo
-/ch/11/mix/03 ON   -oo +0 PRE
-/ch/11/mix/04 ON   -oo
-/ch/11/mix/05 ON  -8.8 +0 PRE
-/ch/11/mix/06 ON  -8.8
+/ch/11/mix/03 ON   0.0 +0 PRE
+/ch/11/mix/04 ON   0.0
+/ch/11/mix/05 ON   -oo +0 PRE
+/ch/11/mix/06 ON   -oo
 /ch/11/mix/07 ON   -oo +0 PRE
 /ch/11/mix/08 ON   -oo
 /ch/11/mix/09 ON   -oo +0 PRE
 /ch/11/mix/10 ON   -oo
-/ch/11/mix/11 ON  -2.3 +0 POST
-/ch/11/mix/12 ON  -2.3
+/ch/11/mix/11 ON  -3.5 +0 POST
+/ch/11/mix/12 ON  -3.5
 /ch/11/mix/13 ON   -oo +0 POST
-/ch/11/mix/14 ON   0.0
+/ch/11/mix/14 ON -39.0
 /ch/11/mix/15 ON   -oo +0 POST
 /ch/11/mix/16 ON   -oo
 /ch/11/grp %00000010 %000000
 /ch/11/automix OFF  +0.0
-/ch/12/config "E Git" 1 GN 12
+/ch/12/config "Synth Bass" 1 GN 12
```

Comment:
Can we call this misc and use default settings (EQ neutralized as well as Dynamics)? You are the only one with the Synth Bass and you might not play it that often, but we will have one additional instrument from time to time. Saving the setting for the synth bass as a preset might do the job.

Reply:
Forget it. I think we just have a Synth Bass and in the rare cases we have another instrument, we just use this or another channel for it.